Resources/views/Block/block_search_result.html.twig

```diff
@@ -12,43 +12,48 @@ file that was distributed with this source code.
 {% extends sonata_block.templates.block_base %}
 
 {% block block %}
-    <div class="col-md-3">
-        <div>
-            <h4>
-                {% if admin.hasRoute('list') %}
-                    <a href="{{ admin.generateUrl('list') }}">{{ admin.label|trans({}, admin.translationdomain) }}</a>
-                {% else %}
-                    {{ admin.label|trans({}, admin.translationdomain) }}
-                {% endif %}
-
-                {% if pager and pager.getNbResults() > 0 %}
-                    <span class="label label-info">{{ pager.getNbResults() }}</span>
-                {% endif %}
-            </h4>
-        </div>
-
-        <ul>
-            {% if pager %}
-                {% for result in pager.getResults() %}
-                    {% if admin.hasRoute('edit') %}
-                        <li><a href="{{ admin.generateObjectUrl('edit', result) }}">{{ admin.toString(result) }}</a></li>
-                    {% else %}
-                        <li><i>{{ admin.toString(result) }}</i></li>
+    <div class="col-lg-4 col-md-6">
+        <div class="box">
+            <div class="box-header with-border">
+                {{ icon|raw }}
+                {{ admin.label|trans({}, admin.translationdomain) }}
+                <h3 class="box-title">
+                    {% set icon = settings.icon|default('') %}
```

Comment:
I think you made a mistake here. icon variable should be before icon rendering and admin label on title, isn't it?